changelog.d/16504.bugfix

```diff
@@ -0,0 +1 @@
+Fix http/s forward proxy authentication not working.
```

Comment:
```suggestion
Fix a bug introduced in Synapse 1.41 where HTTP(S) forward proxy authorization would fail when using basic HTTP authentication with a long `username:password` string.
```